inst/extdata/mvp/product-carbon-transition-risk_review.Rmd

```diff
@@ -0,0 +1,308 @@
+  mutate(score_all = case_when(
+    perc_all < 0.3 ~ "low",
+    perc_all >= 0.3 & perc_all < 0.7 ~ "medium",
+    perc_all >= 0.7 ~ "high",
+  ))
+
+data_short <- data_short %>%
+  mutate(score_unit = case_when(
+    perc_unit < 0.33 ~ "low",
+    perc_unit >= 0.3 & perc_unit < 0.7 ~ "medium",
+    perc_unit >= 0.7 ~ "high",
+  ))
+
+data_short <- data_short %>%
+  mutate(score_unit_isic = case_when(
+    perc_unit_isic < 0.3 ~ "low",
+    perc_unit_isic >= 0.3 & perc_unit_isic < 0.7 ~ "medium",
+    perc_unit_isic >= 0.7 ~ "high",
+  ))
+
+# show distributions (each score should have same probability for score_all)
+barplot(prop.table(table(data_short$score_all)))
+barplot(prop.table(table(data_short$score_unit)))
+barplot(prop.table(table(data_short$score_unit_isic)))
+```
+
+### Step 2: Aggregating on the company-level
+
+      Aggregating on the company-level the percentage of transistion risk products for each company.
+      Calculate 'Product Carbon TR Indicator' for example companies from Hanish's compiled dataset
+
+## Data
+
+## 1. Loading sample company data set & create sample product list
+
+    NOTE: 
+
+    - Problem A with dataset: in the MVP_company_sample dataset the activity names from ecoinvent have an underscore "\_" instead
+      of space in their name. E.g., there is an activity called "treatment_of_tyre_wear_emissions_passenger_car" instead of            "treatment of tyre wear emissions, passenger car" as in the original dataset.
+
+    - Quick solution for now (I don't want to invest time in replacing underscores with spaces and vice versa: I replace the            products with random example from ecoinvent
+
+    - Problem B with dataset: in the sample MVP dataset, some company_name were associated with multiple company_ids (example:          instead of ID number 00001 for each row, excel filled in 00002, 00003, 00004 in the following rows). I adjusted that            manually, i.e. the issue is resolved in the following dataset.
+
+``` {r}
+# load sample data
+data_companies <- params$input2 %||% 
+  read_csv(pctr_path("input_data", "MVP_sample_dataset.csv"))
+
+# select relevant columns
+data_companies <- data_companies %>%
+  select(all_of(c(
+    "Company_ID", "Company_Name", "Products", "activity", "unit_of_product"
+  )))
+
+column_names <- c("company_id", "company_name", "products", "activity", "unit")
+names(data_companies)[c(1, 2, 3, 4, 5)] <- column_names
+
+# select random sample of unique activities, including their unit (uniqueness is
+# for now important because it will make the inner_join easier) two variables
+# sub sample
+sub_sample <- data_short %>%
+  distinct(activity, .keep_all = TRUE)
+
+# random selection of 82 activities incl. unit
+sub_sample <- sub_sample[sample(c(1:nrow(sub_sample)), size = 82, replace = TRUE), c("activity", "unit")]
+
+# replace activity list with new sample from above
+data_companies$activity <- sub_sample$activity
+data_companies$unit <- sub_sample$unit
```

Comment:
Here ends the process of creating `data_companies` -- which is a crucial input of the rest of the process. But `data_companies` should be created outside this main script and imported as a demo dataset. Work on it in data-raw/data_companies.R.

Note some columns have very weird characters that may be problematic. Maybe best to exclude them from the main demo dataset. We can use weird data to test edge cases.

![image](https://user-images.githubusercontent.com/5856545/227340869-7239113d-e0bf-487a-be94-4c7225776c20.png)